#
# the tag ID number for an RFID tag
#
Header header
string rfidTag
string goalOrProximity
